websites/F/Free Web Novel/dist/metadata.json

```
@@ -0,0 +1,96 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "BigBoi©",
```

you're not copyrighted
```suggestion
		"name": "BigBoi",
```